### PR TITLE
KO: translation for site/ko/guide/keras/custom_callback.ipynb

### PR DESCRIPTION
**Pull request by ghdalsrl0708**
Korean translation for custom_callback.ipynb
EN -> KO

EN: https://github.com/tensorflow/docs/blob/master/site/en/guide/keras/custom_callback.ipynb

### CODE REVIEW

File: site/ko/guide/keras/custom_callback.ipynb [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "1OykC-6lI4gv"
+      },
+      "source": [
+        "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://github.com/tensorflow/docs/blob/master/site/en/guide/distribute_strategy.ipynb)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃허브 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 참여하려면 [docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로 메일을 보내주시기 바랍니다.",
+        "\n",
+        "# 케라스 커스텀 callbacks\n",
+        "커스텀 callback은 케라스 모델을 읽고 변경하는 것을 포함하여 교육,평가 또는 추론 중에 케라스 모델의 동작을 사용자 정의를 하는 강력한 도구 입니다. 예시로는 트레이닝 과정과 결과를 텐서보드를 통해 표현하고 시각화 하는 텐서플로우.callbacks.텐서보드 또는, 모델이 트레이닝중에 또는 그 이상에서 자동적으로 저장되는 텐서플로우.callbacks.모델.Checkpoint를 포함합니다. 이안내서에서는 케라스 callback이 무엇인지, 언제 불릴지, 무엇을 할 수 있는지, 그리고 어떨게 여러분 자신의 callback을 시작하기 위해 몇 가지 간단한 callback 애플리케이션을 생성하는 데모가 있을 것입니다."
```
Comment (hongsukkim) [inline site/ko/guide/keras/custom_callback.ipynb]: Please change translation terms: 
'커스텀' -> '사용자 정의'
'callbacks' -> '콜백'
'교육' -> '훈련'
'트레이닝' -> '훈련'

Please do not translate API names:
'텐서플로우.callbacks.텐서보드' -> 'tf.keras.callbacks.TensorBoard'
'텐서플로우.callbacks.모델.Checkpoint' -> 'tf.keras.callbacks.ModelCheckpoint '

Please use word spacing correctly: ' 이안내서' -> '이 안내서'
Please use correct spelling: '어떨게' -> '어떻게'

File: site/ko/guide/keras/custom_callback.ipynb [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "d5zZ8rZD69VW"
+      },
+      "source": [
+        "## 배열"
```
Comment (hongsukkim) [inline site/ko/guide/keras/custom_callback.ipynb]: Please translate 'Setup' to "설치"

File: site/ko/guide/keras/custom_callback.ipynb [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+        "colab": {},
+        "colab_type": "code",
+        "id": "7BazS4qD6-2n"
+      },
+      "outputs": [],
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "try:\n",
+        "  # %코랩에만 존재하는 텐서플로우_버전.\n",
```
Comment (hongsukkim) [inline site/ko/guide/keras/custom_callback.ipynb]: Please do not translate commands: '텐서플로우_버전' -> 'tensorflow_version'

**Comment by lamberta** [comment]
Thank you.

Community translations are now located in the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) repo (see [this message](https://groups.google.com/a/tensorflow.org/g/docs/c/GJPq7S_fw4c/m/lA1qfBX2EQAJ) for context).

I'll move over/duplicate existing pull requests as I find time. In the meantime, please feel free to create a new pull request in that repo 😀 Then you can close this PR and add a link to the new one.

**Comment by lamberta** [comment]
Migrated in https://github.com/tensorflow/docs-l10n/pull/85

**Pull request closed without merging**